tests/e2e_test/module_deletion_test.go

```diff
@@ -0,0 +1,180 @@
+package e2e_test
+
+import (
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	v2 "github.com/kyma-project/lifecycle-manager/internal/declarative/v2"
+	"github.com/kyma-project/lifecycle-manager/pkg/testutils"
```

Comment:
you can import this `testutils` implicitly, to avoid use function with the package name, anyway, the testutils are used for testing.
```suggestion
	. "github.com/kyma-project/lifecycle-manager/pkg/testutils"
```